bundler/lib/bundler/cli.rb

```diff
@@ -753,6 +756,38 @@ def self.reformatted_help_args(args)
       end
     end
 
+    def self.check_deprecated_ext_option(arguments)
+      # when deprecated version of `--ext` is called
+      # print out deprecation warning and pretend `--ext=c` was provided
+      if deprecated_ext_value?(arguments)
+        SharedHelpers.major_deprecation 2, "Option `--ext` without explicit value is deprecated. Please pass value like `--ext=c` for C extension. Pretending `--ext=c` was used for now."
```

Comment:
I'll appreciate any suggestion of better text in here. pray

Reply:
```suggestion
        SharedHelpers.major_deprecation 2, "Extensions can now be generated using C or Rust, so `--ext` with no arguments has been deprecated. Please select a language, e.g. `--ext=rust` to generate a Rust extension. This gem will now be generated as if `--ext=c` was used."
```

Reply:
I'll use this for the final one (other PR onboarding Rust gem generator).